server/dive_server/views_rpc.py

```diff
@@ -41,9 +42,10 @@ def run_pipeline_task(self, folder, pipeline: PipelineDescription):
     @autoDescribeRoute(
         Description("Run training on a folder")
         .jsonParam(
-            "folderIds",
+            "body",
             description="Array of folderIds to run training on",
             paramType="body",
+            schema= {"folderIds": List[str], "labelText": str}
```

Comment:
@subdavis @BryonLewis  Roni and I tried intentionally breaking this by changing the "labelText" to type int and did not get expected results (Nothing broke). Is this the proper way to define the schema?

Reply:
Per a conversation with Brandon, I now understand this schema is for swagger informational purposes, not type checking.